README.md

```diff
@@ -59,12 +59,21 @@ Bump new version number in `CHANGELOG.md` and `README.md`.
 Commit, tag and create a new release:
 
 ```sh
+version="5.0.7"
+
+git switch --create release/${version}
 git add CHANGELOG.md README.md
-git commit -m "Bump v5.0.7"
-git tag v5.0.7
-git push origin master
-git push --tags
-gh release create v5.0.7
+git commit -m "Bump v${version}"
+git push --set-upstream origin release/${version}
```

Comment:
question: is it a required command? I don't remember doing this one before using `gh pr create ...`

Reply:
It's indeed not mandatory. `gh pr create` will do it if needed. I wanted it to be explicitly done in this procedure. That's how we documented it in other open source projects.